src/app/containers/Image/index.jsx

```diff
@@ -36,6 +36,9 @@ const ImageContainer = ({ blocks, position }) => {
   const altTextBlock = filterForBlockType(blocks, 'altText');
   const captionBlock = filterForBlockType(blocks, 'caption');
 
+  const ShouldPreLoadLeadImage =
+    position[0] <= LAZYLOAD_FROM_BLOCK && shouldPreload;
```

Comment:
- I have a more general question: is the first `image` block in a page always the lead?
- Is there a case where we could preload three images if they have positions less than four/`LAZYLOAD_FROM_BLOCK`? e.g.:

```js
{
  blocks: [
    heading,
    image, // Position 2, Lead, pre-loaded.
    image, // Position 3, Pre-loaded?
    image, // Position 4, Pre-loaded?
    text,
    image, // Position 5, Lazy-loaded.
  ]
}
```

Reply:
I'd rather this logic wasn't positional but based on some block metadata saying the image is a 'lead image', @jimjohnsonrollings / @aeroplanejane does such metadata exist, I suspect not?

Reply:
There is no such concept in Optimo, and I don't think there is one in CPS.

Reply:
@andrewscfc, I agree.

Can I suggest we decorate the incoming block ourselves? There is a precedence for this with Social Embeds, as we needed a way of working out their position in the page to create unique skip-links. e.g. `#tweet-1`, `#tweet-2`.

Reply:
@HarveyPeachey and @chewu looked into implementing this in this PR https://github.com/bbc/simorgh/pull/8807 but the solution looks to be pretty complicated as the optimo block structure needs to be modified, in my opinion we'd be better tackling this in a seperate issue or just leaving it. Going to approve the PR on that basis, others feel free to discuss if you feel it would be better tackled in this PR.